### DISCUSSION TITLE
favorite list resets every time the bot restarts

### DISCUSSION THREAD

**Discussion by xKosar** [codetheweb/muse#1038] [Q&A]
is this a bug or is it normal for the favorite list to be erased every time the host restarts the bot? is there no way to save favorite list?
![Screenshot_1](https://github.com/codetheweb/muse/assets/78801680/7846d0c3-b2e5-4536-b173-58ab9519a5fb)

**Comment by codetheweb** [comment]
If you're using Docker, maybe make sure that the data directory is being saved in a volume or host directory across reboots?
